i18n/de/middleware/README.md

```diff
@@ -997,28 +997,48 @@ Folgende Systemvariablen stehen Ihnen zur Verwendung im Template zur Verfügung:
-| `reference_scu_dieboldnixdorf_counter_{0-n}_id`  | wird verwendet um eine bereits vorhandene diebold SCU zu referenzieren, die in ihrer Konfiguration einen Counter gesetzt hat      |
-| `reference_scu_epson_counter_{0-n}_id`           | wird verwendet um eine bereits vorhandene epson SCU zu referenzieren, die in ihrer Konfiguration einen Counter gesetzt hat        |
-| `reference_scu_atrust_counter_{0-n}_id`          | wird verwendet um eine bereits vorhandene atrust SCU zu referenzieren, die in ihrer Konfiguration einen Counter gesetzt hat       |
-| `reference_scu_fiskaly_counter_{0-n}_url`        | wird verwendet um die Url einer bereits vorhandenen fiskaly cloud SCU zu erhalten                                                 |
-| `reference_scu_swissbitcloud_counter_{0-n}_url`  | wird verwendet um die Url einer bereits vorhandenen Swissbit cloud SCU zu erhalten                                                |
-| `reference_scu_swissbit_counter_{0-n}_url`       | wird verwendet um die Url einer bereits vorhandenen Swissbit usb SCU zu erhalten                                                  |
-| `reference_scu_cryptovision_counter_{0-n}_url`   | wird verwendet um die Url einer bereits vorhandenen cryptovision SCU zu erhalten                                                  |
-| `reference_scu_dieboldnixdorf_counter_{0-n}_url` | wird verwendet um die Url einer bereits vorhandenen diebold SCU zu erhalten                                                       |
-| `reference_scu_epson_counter_{0-n}_url`          | wird verwendet um die Url einer bereits vorhandenen epson SCU zu erhalten                                                         |
-| `reference_scu_atrust_counter_{0-n}_url`         | wird verwendet um die Url einer bereits vorhandenen atrust SCU zu erhalten                                                        |
+| `cashbox_id`                                     | Zufällige GUID                                                                                                                    |
+| `scu{0-9}_id`                                    | Zufällige GUID                                                                                                                    |
+| `helper{0-9}_id`                                 | Zufällige GUID                                                                                                                    |
+| `queue{0-9}_id`                                  | Zufällige GUID                                                                                                                    |
+| `queue{0-9}_id_base64withoutspecialchars`        | `{queue_id}`, konvertiert zu Base64 ohne Sonderzeichen. Wird oft als Kassenseriennummer verwendet (CashBoxIdentification)         |
+| `reference_scu_{type}_{key}_{value}_{attr}`      | Referenziert auf das angegebene Attribut einer existierenden SCU des gegebenen Typs und Schlüssel/Werte Paars                     |
```

Comment:
Vielleicht hinzufügen, dass die dynamischen Werte unten genauer beschrieben werden (auch im englischen Teil)

Reply:
Can you elaborate? 🙂